fantasy_football_app/forms.py

```diff
@@ -1,11 +1,50 @@
 # forms.py
 from django import forms
+from django.contrib.auth.forms import AuthenticationForm, UserCreationForm
+from django.contrib.auth.models import User
 
 from .constants import FLEX_POSITIONS
 from .custom_field_choices import (GroupedModelChoiceField,
                                    get_custom_grouped_model_choice_field)
 from .models import Entry, Player, RosteredPlayers
 
+# -------------------------------------------------------------------------
+# ----- authentication forms -----
+# -------------------------------------------------------------------------
+
+class RegistrationForm(UserCreationForm):
```

Comment:
I think these were the legacy forms that we won't use anymore. feel free to just delete them. no need to keep these gross forms aroudn